locale/en/download/package-manager.md

```diff
@@ -289,6 +290,28 @@ Or install a binary package (if available for your platform) using pkgin:
 pkgin -y install nodejs
 ```
 
+## nvm
+On Unix / OS X systems Node.js built from source can be installed by
+[nvm](https://github.com/creationix/nvm) by installing into the location that nvm expects:
+
+```bash
+$ make install DESTDIR='$(NVM_DIR)/versions/node/' PREFIX='$(FULLVERSION)-pre'
```

Comment:
I don't think that we would want to source `nvm.sh` from the Makefile. I think the Makefile should stay unchanged and not have anything nvm specific. 

I've looked into exporting the function but this looks like it would not be supported by all shells. 

I'm suggesting that we change this to be `DESTDIR=~/.nvm/versions/node/` and then document that if nvm has been installed into a different location then the path must be updated to reflect that.

What do you think about that?